tests/e2e-leg-2/use-own-serviceaccount/setup-vdb/base/setup-vdb.yaml

```diff
@@ -0,0 +1,33 @@
+# (c) Copyright [2021-2023] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertica.com/v1beta1
```

Comment:
Couldn't we also test that if the sa doesn't exist but the role and rolebinding do then only the sa is created with the specified name?

Reply:
The operator cannot find user created role/rolebinding. So, if the sa is deleted the operator will generate that along with new role/rolebindings. I'll expand this test to include that scenario.